### ISSUE TITLE
`woeusbgui` aborted with "Fatal Error: Mismatch between the program and library build versions detected."

### ISSUE THREAD

**Issue by Lin-Buo-Ren** [slacka/WoeUSB#105]
## Good Habit Checklist for Issue Reporting
* [x] I've searched the issue tracker and is pretty sure that there's no duplicate issue already filed.

## Issue Reproduce Instructions
1. Launch WoeUSB by running `woeusbgui`

## Expected Behavior
WoeUSB launched without issues

## Current Behavior
WoeUSB aborted with:

```
Fatal Error: Mismatch between the program and library build versions detected.
The library used 2.8 (no debug,Unicode,compiler with C++ ABI 1009,wx containers,compatible with 2.6),
and your program used 2.8 (no debug,Unicode,compiler with C++ ABI 1002,wx containers,compatible with 2.6).
```

### WoeUSB Version
2.0.8 from the releases page

### OS Distribution
Ubuntu 16.04 AMD64

**Comment by Lin-Buo-Ren** [comment]
This is because WxWidgets checks the library version and refuses to run if mismatch, the build on Travis CI is on Ubuntu 14.04 while this system is 16.04, thus the package is incompatible with it.

Commit 029652d112219f7f2b3e66f5e27ef7f522153c6e disables the deployment of the built package form Travis CI, this issue should be fixed by #104 instead.

**Comment by Askmewho** [comment]
the sames happend on rpi3! 
how to solve that?
